src/device.coffee

```diff
@@ -128,13 +128,11 @@ exports.getDeviceType = do ->
 					throw new Error('Device type not specified in config file')
 				return configFromFile.deviceType
 
-# Calling this function updates the local device state, which is then used to synchronise
-# the remote device state, repeating any failed updates until successfully synchronised.
-# This function will also optimise updates by merging multiple updates and only sending the latest state.
-exports.updateState = do ->
+do ->
 	applyPromise = Promise.resolve()
 	targetState = {}
 	actualState = {}
+	updatePending = false
```

Comment:
isn't this the same variable as updateStatus.state in src/application.coffee? :/